graph/number_of_distinct_islands.cpp

```diff
@@ -0,0 +1,120 @@
+#include <iostream>
+#include <vector>
+#include <set>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```